micrometer-core/src/main/resources/META-INF/native-image/io.micrometer/micrometer-core/reflect-config.json

```diff
@@ -27,5 +27,71 @@
   {
     "name":"org.HdrHistogram.Histogram",
     "methods":[{"name":"<init>","parameterTypes":["long","long","int"] }]
+  },
+  {
+    "name":"com.hazelcast.core.DistributedObject",
+    "methods":[{"name":"getName","parameterTypes":[] }]
```

Comment:
I think it's going to get unwieldy quickly if we try to put reflection config for everything we have instrumentation for here. Since there is already metadata [here](https://github.com/oracle/graalvm-reachability-metadata/tree/master/metadata/com.hazelcast/hazelcast), can users not rely on that? That way we don't have to maintain reflection config for Hazelcast and distribute it in our core jar when most of our users may not even use hazelcast?

Reply:
@shakuzen Thanks for the feedback!

I'm not sure if I understand your feedback correctly, but these reflection hints are due to reflection usages on Hazelcast in the `HazelcastIMapAdapter`, so I thought this was a proper place for them.

Am I missing something?

Reply:
I think I just wasn't paying full attention. That makes sense, and I guess there isn't a good alternative to us including the reflection config here, then.